promql/engine.go

```diff
@@ -1676,7 +1676,7 @@ func (ev *evaluator) VectorBinop(op parser.ItemType, lhs, rhs Vector, matching *
 				insertedSigs = map[uint64]struct{}{}
 				matchedSigs[sig] = insertedSigs
 			} else if _, duplicate := insertedSigs[insertSig]; duplicate {
-				ev.errorf("multiple matches for labels: grouping labels must ensure unique matches")
+				ev.errorf("multiple matches for labels: grouping labels must ensure unique matches: duplicate metric %s found as a result of the operation [%s, %s]", metric.String(), ls.Metric.String(), rs.Metric.String())
```

Comment:
Duplicate output series. 

It's also confusing to say "operation" when the operation has no relevance, as the logic is the same for all the binops that hit this code.

Reply:
You're saying that the output series will be printed twice?

Will rephrasing it to say "as a result of the binary operation between []" be enough to avoid this potential source of confusion?

Reply:
I'm saying that "duplicate metric" is not the best wording, "duplicate output series" is better.

Yes, though something shorter is probably also possible.

Reply:
Thanks for the clarification @brian-brazil